ngraph/python/tests/test_ngraph/test_manager.py

```diff
@@ -0,0 +1,61 @@
+# ******************************************************************************
+# Copyright 2017-2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ******************************************************************************
+# flake8: noqa
+import json
+
+import numpy as np
+import pytest
+
+import ngraph as ng
+from ngraph.impl import Function, PartialShape, Shape
+from ngraph.impl.passes import Manager
+from tests.test_ngraph.util import count_ops_of_type
+from ngraph.utils.types import (
+    NodeInput,
+    NumericData,
+    NumericType,
+    ScalarData,
+    TensorShape,
+    as_node,
+    as_nodes,
+    get_dtype,
+    get_element_type,
+    get_element_type_str,
+    make_constant_node,
+)
+
+
+def test_constant_folding():
+    node_constant = ng.constant(np.array([[0.0, 0.1, -0.1], [-2.5, 2.5, 3.0]], dtype=np.float32))
+    node_ceil = ng.ceiling(node_constant)
+    func = Function(node_ceil, [], "TestFunction")
+
+    assert count_ops_of_type(func, node_ceil) == 1
+    assert count_ops_of_type(func, node_constant) == 1
+
+    pass_manager = Manager()
+    pass_manager.register_pass("ConstantFolding")
+    pass_manager.run_passes(func, None)
```

Comment:
Why you pass None  here?

Reply:
As far as I remember the problem, run_passes function require two arguments in C++ API. Pybind won't accept call without the second argument which in C++ is omitted - this None value was a easy way to get workaround. I will look into this today and confirm it. However pybind has it's limitations which require tricks to get desired results.

I think the easiest (and probably only?) way would be overriding Manager class on Python side in the similar way that [PR1696](https://github.com/openvinotoolkit/openvino/pull/1696) does (overriding __setitem__ method in RTMap).

Reply:
This second argument is not used and I think we need to remove it from C++.

Reply:
Just do it in this PR?

Reply:
I guess it's small change and I would suggest to make it in this PR